arrow/src/main/java/org/apache/calcite/adapter/arrow/ArrowTranslator.java

```diff
@@ -127,6 +127,8 @@ private String translateMatch2(RexNode node) {
       return translateBinary("greater_than", "<", (RexCall) node);
     case GREATER_THAN_OR_EQUAL:
       return translateBinary("greater_than_or_equal_to", "<=", (RexCall) node);
+    case IS_NULL:
+      return translateUnary("isnull", (RexCall) node);
     default:
       throw new UnsupportedOperationException("Unsupported binary operator " + node);
```

Comment:
I realize now that "binary" is not correct, it can be unary too, can we amend as `"Unsupported operator " + node`?